.devcontainer/Dockerfile

```diff
@@ -1 +1 @@
-FROM bkuhlmann/alpine-ruby:latest
```

Comment:
:bulb: The reason for defaulting to this image is because the footprint is smaller due to using Alpine Linux. We can use `ruby:latest`, makes sense, but might want to revisit this in the future.

Reply:
I understand we want it to be small, but we specifically can't use an alpine based distro if we also want it to use the github-cli tools.

Reply:
Wait, what do you mean by GH CLI tools? Do you mean GH's official client for making commits? I use pure Git from the command line only so not familiar with how that tool works but would like to learn more especially how that CLI would have issue with running a Docker container like this.